src/Annotation/EloquaAppCloudContentResponder.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\eloqua_app_cloud\Annotation;
+
+use Drupal\Component\Annotation\Plugin;
+
+/**
+ * Defines a Eloqua AppCloud Content Responder item annotation object.
+ *
+ * @see \Drupal\eloqua_app_cloud\Plugin\EloquaAppCloudContentResponderManager
+ * @see plugin_api
+ *
+ * @Annotation
+ */
+class EloquaAppCloudContentResponder extends Plugin {
+
+
+  /**
+   * The plugin ID.
+   *
+   * @var string
+   */
+  public $id;
+
+  /**
+   * The label of the plugin.
+   *
+   * @var \Drupal\Core\Annotation\Translation
+   *
+   * @ingroup plugin_translatable
+   */
+  public $label;
+
+  /**
+   * The list of fields required by the plugin.
+   *
+   * @var array
+   *
+   */
+  public $fieldList;
+
+  /**
+   * The API type (contacts or customObject).
+   * @var string
+   */
+  public $api;
+
+
+  /**
+   * The name of the queue worker this plugin requires.
+   * @var string
+   */
+  public $queueWorker;
```

Comment:
Is it actually necessary to expose this as a plugin config?  Seems like there's a 1:1 relationship between plugin and queue worker.